### DISCUSSION TITLE
Allow for drag and drop of more than one issue between columns in views

### DISCUSSION THREAD

**Discussion by conatus** [community/community#19594] [Projects and Issues]
Hi all,

As far as I can see this is not being tracked

At the moment when I select multiple issues and try and move them between columns in a view, then it only moves the first issue I have selected.

When planning out cycles, this means I have to move issues one by one and this is quite time consuming as you might imagine.

Thanks in advance!

**Comment by conatus** [comment]
Should be labelled with [Projects](https://github.com/github-community/community/discussions?discussions_q=label%3AProjects)

**Comment by conatus** [comment]
Also unsure if this is a bug!

**Comment by bartecargo** [comment]
It seems very counter-intuitive that selecting multiple issues and dragging them only affects one issue.

**Comment by conatus** [comment]
Does anyone have a workaround for this?

Feels very counterintutive I spend so much of my day moving things one by one into different columns!

**Comment by conatus** [comment]
Potential duplicate of https://github.com/community/community/discussions/11863
